README.md

```diff
@@ -83,6 +83,55 @@ In the index.html, include the core branding stylesheet:
 #### **avatar**
 `string` Avatar image URL. Default is the special provided Antwerp icon.
 
+### Events
+
+#### **actionClicked**
+Triggers when the user clicks a button of a message with type `action`. This type of message can only be generated by the BFF package. The `actionClicked` output event requires a defined action and pauses the workings of the chatbot widget, waiting for a method call (`completeAction`) that has the same defined action. It allows the developer to handle some things outside the chatbot (e.g. a payment) before returning to the chatbot conversation.
+
+##### How does it work?
+
+```
+<aui-chatbot
+  #myChatbotWithAction
+  [url]="'http://localhost:3000/api/chatbot'"
+  [session]="session1"
+  (actionClicked)="triggerMe($event)">
+</aui-chatbot>
+```
+
+1. Your BFF sends a message of type `action` with one or more actions defined.
+
+  ```
+  {

```

Comment:
The format of a message is always the same this is a snippet from the response is this what you are documenting?
```js

{
  "type":"action",
  "message":"message to show in button",
  "action":"foo",
  "url":null,
  "image":null
}
```

Reply:
Fixed in the way we discussed.

Reply:
```js
    actions: [
        {
            text: "test for button",
            action: "foo"
            params: { foo: "bar" } // optional
        },
    ]
```